.flake8

```diff
@@ -1,3 +1,5 @@
 [flake8]
-max-line-length = 120
-ignore = D100,W503,E231
+extend-ignore = E133, E203, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503
+max-line-length = 110
+max-doc-length = 79
+exclude = __init__.py
```

Comment:
Since this file is automatically generated with exactly the same contents using `generate_pre_commit_config`, please remove it.